pkg/kubelet/kubelet_test.go

```diff
@@ -1993,6 +2009,14 @@ func TestHandlePodResourcesResize(t *testing.T) {
 	defer kubelet.podManager.DeletePod(testPod2)
 	defer kubelet.podManager.DeletePod(testPod1)
 
+	kubeclientmanager.NewKubeClientManager()
+	kubeclientmanager.ClientManager.RegisterTenantSourceServer(
+		"api",
```

Comment:
Let's use the constant instead of hard-coded "api".

Reply:
fixed